Moq.AutoMock/AutoMocker.cs

```diff
@@ -14,17 +14,39 @@ namespace Moq.AutoMock
     public partial class AutoMocker
     {
         private readonly Dictionary<Type, IInstance> typeMap = new Dictionary<Type, IInstance>();
-        private readonly MockBehavior mockBehavior;
+
+        public AutoMocker()

```

Comment:
Rather than each constructor duplicating code. It would be better to have each constructor call the one below it passing in the default values. This keeps the defaults in one location.

Reply:
Good call.  I'll make it so.